core/src/main/java/org/stellar/anchor/sep24/Sep24Service.java

```diff
@@ -212,11 +212,13 @@ public InteractiveTransactionResponse withdraw(
             .sep10Account(token.getAccount())
             .sep10AccountMemo(token.getAccountMemo())
             .fromAccount(sourceAccount)
-            // TODO - jamie to add unique address generator
-            .withdrawAnchorAccount(asset.getDistributionAccount())
             .toAccount(asset.getDistributionAccount())
             .clientDomain(token.getClientDomain());
 
+    if (asset.getDistributionAccount() != null) {
```

Comment:
It is safer to use `isEmpty()`

Reply:
Wouldn't it throw npe?

Reply:
`StringHelper.isEmpty()` won't throw npe.